protocol/scripts/governance/community_treasury_sending.py

```diff
@@ -0,0 +1,67 @@
+import csv
+import decimal
+import json
+import sys
+
+# Note: value assumes that the token exponent is 18 (i.e. 1 full token = 1 * 10^18 native token denom
+TOKEN_MULTIPLE = decimal.Decimal(1000000000000000000)
+
+
+# Usage:
+# 1. Update below section with appropriate values
+# 2. Run `python3 community_treasury_sending.py <csv file>`.
+#    Each line should have two fields: recipient address and number of full tokens to send.
+# 3. Use generated `.json` file to submit proposal.
+
+########################################################################## 
+### BEGIN: Required proposal fields.                                   ###
+### TODO: update below fields as needed                                ###
+########################################################################## 
+TITLE="TODO: Fill in proposal title" 
+NATIVE_TOKEN_DENOM="adv4tnt" # TODO: Replace with production token 
+PROPOSAL_BODY="""
+TODO: Fill in proposal content. 
```

Comment:
Nit: As of `0.47` there was an [issue](https://dydx-team.slack.com/archives/C05SY5VJL1E/p1701098286713659?thread_ts=1700833286.301049&cid=C05SY5VJL1E) where `summary` field is quite short. Can you change this to something like "Include a brief summary of the proposal and link to relevant governance forum discussion"